docs/bmad/stories/48.10.layout-modi.md

```
@@ -0,0 +1,976 @@
+   - **NEW:** Vollbild-Command im Command Palette
+   - Added `useMatchRoute` import
+   - Added `PiArrowsOut` icon import
+   - Added `isOnKarteRoute` check (matchRoute für /karte)
+   - Added "Vollbildmodus aktivieren" zu subPages (nur wenn auf /karte Route)
+   - Keyboard Shortcut: `⌘ + E` (Cmd+E)
+   - Navigation mit search parameter update
+   - Added `isOnKarteRoute` zu useMemo dependencies
+
+**Neue Dateien:**
+
+6. ✅ `packages/frontend/src/components/organisms/lagekarte/FullscreenCloseButton/FullscreenCloseButton.tsx`
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)